packages/swirl-components/src/components/swirl-popover-trigger/swirl-popover-trigger.tsx

```diff
@@ -80,6 +82,8 @@ export class SwirlPopoverTrigger {
 
     popoverEl.open(triggerEl, true);
 
+    this.stopHoverLingerDelay();
```

Comment:
Nitpick: componentDidLoad is only called once at the beginning of the component instance lifecycle. There should be no need to stop the delay handler here.

Reply:
it's called from mouseEnter not in componentDidLoad

Reply:
Ah, the code folding of Github confused me.